src/language-html/printer-html.js

```diff
@@ -54,15 +54,16 @@ function genericPrint(path, options, print) {
       return htmlWhitespaceUtils.trim(node.expression);
 
     case "angularLetDeclaration":
-      // basically printed via embedded formatting
-      return [
+      // print like "break-after-operator" layout assignment in estree printer
+      return group([
         "@let ",
-        node.name,
-        " = ",
-        node.value,
+        group([node.id, " =", group(indent([line, print("init")]))]),
         // semicolon is required
         ";",
-      ];
+      ]);
```

Comment:
Move the layout part into this, so we can have a better break even `--embedded-language-formatting=off`